src/lava/magma/runtime/runtime_service.py

```diff
@@ -91,15 +91,19 @@ class Phase:
         POST_MGMT = enum_to_np(4)
         HOST = enum_to_np(5)
 
-    def _next_phase(self, curr_phase):
+    def _next_phase(self, curr_phase, is_last_time_step: bool):
```

Comment:
While you are at it, please add docstring and inline comments to explain the Loihi protocol.

Reply:
Done.